src/ilastik-packager.imjoy.html

```diff
@@ -0,0 +1,348 @@
+    compression: "DEFLATE",
+    compressionOptions: {
+        level: 9
+    }},
+    (mdata)=>{
+      showProgress(mdata.percent)
+    }
+  )
+  await api.exportFile(zipBlob, spec.name.replace(/\s+/g, '-').toLowerCase() + '.zip')
+  await showMessage('Model package was exported successfully')
+
+}
+
+const app = new Vue({
+  el: '#app',
+  data: {
+    modelInfo: null,
+    format: null,
+    status: "",
+    loading: false,
+    progress: 0,
+    spec: null,
+    rootUrl: null,
+  },
+  methods: {
+    async init(weight_formats_spec){
+      const rdfFile = this.modelInfo.config&&this.modelInfo.config._rdf_file;
+      let specUrl = rdfFile || this.modelInfo.source
+      const tmp = specUrl.split('/')
+      this.rootUrl = tmp.slice(0, tmp.length-1).join('/')
+      specUrl = await convertZenodoFileUrl(specUrl) || specUrl
+      const response = await fetch(specUrl)
+      this.spec = jsyaml.load(await response.text())
+
+      this.modelInfo.name = this.spec.name
+      this.spec.config = this.spec.config || {}
+      this.spec.config.weights_consumers = Object.keys(this.spec.weights).map((k)=>{
+          const wformat = weight_formats_spec.weight_formats[k]
+          if(wformat)
+            return {"format": k, "name": wformat.name, "consumers": wformat.consumers.map((k)=>weight_formats_spec.consumers[k])}
+          else
+            return {"format": k, "name": k, "consumers": null}
+      })
+
+    },
+    async download(format){
+      this.format = format
+      try{
+        this.loading = true;
+        if(this.format === 'default')
+          await api.utils.openUrl(spec.download_url)
+        else
+          await downloadModel(this.rootUrl, this.spec, this.format, async (msg)=>{
+            this.status = msg
+            this.$forceUpdate()
+          }, async (p)=>{
+            if(p<1) p = parseInt(p*100)
+            this.progress = p
+            this.$forceUpdate()
+          })
+      }
+      finally{
+        this.progress = 0
+        this.loading = false
+      }
+
+    }
+  }
+})
+
+class ImJoyPlugin {
+  async setup() {
+    const response = await fetch("https://raw.githubusercontent.com/bioimage-io/spec-bioimage-io/gh-pages/weight_formats_spec.json")
+    this.weight_formats_spec = JSON.parse(await response.text())
+  }
+
+  async run(ctx) {
+    if(!ctx.data || (!ctx.data.config && !ctx.data.source)){
+      const specUrl = await api.prompt("Please paste the model spec url here", "https://raw.githubusercontent.com/deepimagej/models/master/defcon_density_map_estimation/model.yaml")
+      if(!specUrl) return
+      app.modelInfo = {
+        name: getFileName(specUrl),
+        config: { _rdf_file: specUrl }
+      }
+    }
+    else
+      app.modelInfo = ctx.data;
+    await app.init(this.weight_formats_spec);
+    app.$forceUpdate();
+  }
+}
+
+api.export(new ImJoyPlugin())
+</script>
+
+<window lang="html">
+  <div style="text-align: center;" id="app">
+    <p>Model Packager</p>
```

Comment:
Here, it might be good to insert ilastik banner, something like this:
```suggestion
    <img src="https://www.ilastik.org/assets/ilastik-logo.png" alt="ilastik">
    <h1>
      <a href="https://www.ilastik.org/" target="_blank">
          ilastik
      </a>
    </h1>
    <p>The interactive learning and segmentation toolkit</p>
    <h4>You can download and run this model in ilastik:</h4>
```

This is how it looks like:
<img width="604" alt="Screenshot 2021-06-29 at 11 34 00" src="https://user-images.githubusercontent.com/478667/123774708-f8dc1400-d8cd-11eb-9dba-f1daa3c7bd26.png">